setup.py

```diff
@@ -34,7 +34,7 @@
     packages=find_namespace_packages(where="src", exclude=("test",)),
     package_dir={"": "src"},
     install_requires=[
-        "amazon-braket-sdk>=1.35.0",
+        "amazon-braket-sdk>=1.47.0",

```

Comment:
That's a lotta versions

Reply:
interestingly, tox passed with this

Reply:
Oh that's what you meant, I misread the comment that I left both lines in the file lol